frontend/src/components/Table/Table.tsx

```diff
@@ -264,8 +264,8 @@ const Table = <T extends IIdentifiedObject, TFilter extends object = {}>(
     columns,
     data,
     onRequestData,
-    totalItems,
-    pageCount,
+    totalItems: externalTotalItems,
+    pageCount: externalPageCount,
```

Comment:
I get a strong feeling that the table component should be refactored. Not now ofc

Reply:
Absolutely, it is huge.